build.gradle

```diff
@@ -79,23 +79,23 @@ dependencies {
     compile 'postgresql:postgresql:9.1-901-1.jdbc4'
```

Comment:
The Postgres JDBC driver we're using is _really_ old.  It looks like they changed the group name for their artifacts from `postgresql` to `org.postgresql`.  There's been 74 new artifacts published since this version was released.  The latest version (for Java 8) is `org.postgresql:postgresql:42.1.4`.

Do you want to upgrade that as part of this PR or should it be done separately?

Reply:
Good catch, noticed that as well, but didn't look into any alternative group names!